File: kyw0716/2주차/첫 시도/K번째 수.js
function solution(array, commands) {
  let answer = [];

  commands.forEach((command) => {
    let subArray = array
      .slice(command[0] - 1, command[1])
      .sort((a, b) => a - b);
    answer.push(subArray.at(command[2] - 1));
  });
  return answer;
}

File: sk000801/k번째수.java
import java.util.Arrays;
class Solution {
    public int[] solution(int[] array, int[][] commands) {
        int[] answer = new int[commands.length];
        for(int i=0; i<commands.length; i++) {
            int[] cut = Arrays.copyOfRange(array, commands[i][0]-1, commands[i][1]);
            Arrays.sort(cut);
            answer[i] = cut[commands[i][2]-1];
        }
        return answer;
    }
}